modules/service/src/main/resources/db/migration/V0864__emails.sql

```diff
@@ -0,0 +1,107 @@
+-- Status of an email
+CREATE TYPE e_email_status AS ENUM (
+  'queued',
+  'rejected',
+  'accepted',
+  'delivered',
+  'permanent_failure',
+  'temporary_failure'
+);
+COMMENT ON TYPE e_email_status IS 'Status of emails.';
+
+CREATE TABLE t_email (
+  c_email_id          TEXT           NOT NULL PRIMARY KEY CHECK(length(c_email_id) > 0),
```

Comment:
How is the id structured?  It surprised me to see it can be arbitrary text.  I suppose it comes from the mail service provider.

Reply:
Yes, it comes from the email provider. It has a structure, of sorts, but it seems imprudent to rely on that when we have no control over it and it isn't documented. It looks like `20240411142701.584cf35193edf2d5@gpp.bernmohr.com`. So, currently it is a valid email address.